test/sharness/t0080-repo.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+#
+# Copyright (c) 2014 Jeromy Johnson
+# MIT Licensed; see the LICENSE file in this repository.
+#
+
+test_description="Test ipfs repo operations"
```

Comment:
Im getting a little better at writing them finally, so youll probably see more of those out of me.